mix.exs

```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
-      {:membrane_aac_plugin, "~> 0.13.0"},
-      {:membrane_rtp_h264_plugin, "~> 0.15.1"},
-      {:membrane_rtp_opus_plugin, "~> 0.7.0"},
-      {:membrane_rtp_plugin, "~> 0.21.0"},
-      {:membrane_funnel_plugin, "~> 0.7.0"},
-      # {:membrane_ice_plugin, "~> 0.14.3"},
-      {:membrane_rtp_vp8_plugin, "~> 0.7.1"},
-      {:membrane_opus_plugin, "~> 0.16.0"},
-      # {:membrane_webrtc_plugin, "~> 0.14.2"},
-      {:membrane_aac_fdk_plugin, "~> 0.14.0"},
+      {:membrane_core, "~> 0.12.7"},
+      {:membrane_h264_ffmpeg_plugin, "~> 0.30.0"},
+      {:membrane_file_plugin, "~> 0.15.0"},
+      {:membrane_common_c, "~> 0.15.0"},
+      {:membrane_mp4_plugin, "~> 0.30.1"},
+      {:membrane_tee_plugin, "~> 0.11.0"},
+      {:membrane_aac_plugin, "~> 0.16.0"},
+      {:membrane_rtp_h264_plugin, "~> 0.18.0"},
+      {:membrane_rtp_opus_plugin, "~> 0.8.0"},
+      {:membrane_rtp_plugin, "~> 0.23.0"},
+      {:membrane_funnel_plugin, "~> 0.8.0"},
+      {:membrane_ice_plugin, "~> 0.16.0"},
+      {:membrane_rtp_vp8_plugin, "~> 0.8.0"},
+      {:membrane_opus_plugin, "~> 0.18.0"},
+      {:membrane_webrtc_plugin, "~> 0.16.1"},
+      {:membrane_aac_fdk_plugin, "~> 0.17.0"},
       {:membrane_realtimer_plugin, "~> 0.6.1"},
-      {:membrane_framerate_converter_plugin, "~> 0.6.0"},
-      # {:membrane_video_compositor_plugin, "~> 0.2.1"},
-      {:membrane_generator_plugin, "~> 0.8.0"},
-      # {:membrane_rtc_engine, "~> 0.11.0"},
-      {:membrane_audio_mix_plugin, "~> 0.12.0"},
-      {:membrane_audio_filler_plugin, "~> 0.1.0"},
-      {:membrane_rtmp_plugin, "~> 0.11.2"},
-      {:membrane_flv_plugin, "~> 0.5.0"},
-      {:membrane_ffmpeg_swscale_plugin, "~> 0.11.0"},
-      {:membrane_video_merger_plugin, "~> 0.7.0"},
-      {:membrane_stream_plugin, "~> 0.2.1"},
-      # {:membrane_sdl_plugin, "~> 0.15.0"},
-      {:membrane_ffmpeg_video_filter_plugin, "~> 0.9.0"},
-      {:membrane_ffmpeg_swresample_plugin, "~> 0.16.1"},
-      {:membrane_udp_plugin, "~> 0.9.2"},
-      {:membrane_h264_plugin, "~> 0.2.1"},
-      {:membrane_portaudio_plugin, "~> 0.15.0"},
+      {:membrane_framerate_converter_plugin, "~> 0.7.0"},
+      {:membrane_generator_plugin, "~> 0.9.0"},
+      {:membrane_rtc_engine, "~> 0.17.0"},
+      {:membrane_audio_mix_plugin, "~> 0.15.0"},
+      {:membrane_audio_filler_plugin, "~> 0.2.0"},
+      {:membrane_rtmp_plugin, "~> 0.17.2"},
+      {:membrane_flv_plugin, "~> 0.9.0"},
+      {:membrane_ffmpeg_swscale_plugin, "~> 0.14.0"},
+      {:membrane_video_merger_plugin, "~> 0.8.0"},
+      {:membrane_stream_plugin, "~> 0.3.1"},
+      {:membrane_sdl_plugin, "~> 0.17.0"},
+      {:membrane_ffmpeg_video_filter_plugin, "~> 0.11.0"},
+      {:membrane_ffmpeg_swresample_plugin, "~> 0.18.0"},
+      {:membrane_udp_plugin, "~> 0.10.0"},
+      {:membrane_h264_plugin, "~> 0.7.0"},
+      {:membrane_portaudio_plugin, "~> 0.17.0"},
       {:bunch, "~> 1.6", override: true},
-      {:membrane_mp3_mad_plugin, "~> 0.14.0"},
```

Comment:
we should put back the precompiled MAD for Mac ARM

Reply:
https://github.com/membraneframework/membrane_mp3_mad_plugin/pull/45